README.md

```diff
@@ -1,15 +1,122 @@
 # A Kubernetes Operator for Synapse
 
-This project is in development. The Synapse operator has not been published on
-the Operator Hub yet.
+The Synapse
+[operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+offers a convenient way to deploy and manage a
+[Synapse](https://github.com/matrix-org/synapse/) server. It was built with
+[operator-sdk](https://sdk.operatorframework.io/).
+
+## Deploying the Synapse Operator
+
+Each of the following sections present one way to deploy the Synapse operator.
+
+### Run the controller locally with `make run`
+
```

Comment:
I believe we could create a section for contributors on how to develop and build and include the make targets. Both run and deploy would be part of the development cycle. We can use kustomize to create a single yaml file to install the operator that would be way simpler for people to use, with hard coded release versions that we know are working. That will prevent people that just want to try it from messing up with Make variables. Here is how I did that with snoopy-operator:

```
manifest-package: manifests kustomize
	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
	$(KUSTOMIZE) build config/default | tee config/install/snoopy-operator.yaml
```
I've created a make target to create this install file with each release.

Reply:
I've followed your advise and added this make target.

I don't think a contributors documentation is needed at this step.